components/brave_rewards/resources/ui/components/modalRedirect/index.tsx

```diff
@@ -18,8 +18,8 @@ import { getLocale, splitStringForTag } from '../../../../../common/locale'
 
 export interface Props {
   id?: string
-  errorText?: string
-  errorTextLink?: string
+  errorText?: string[]
+  errorTextLink?: string[]
```

Comment:
I feel like there should only be one `errorTextLink`. Making it an array seems like overkill, unless there's a specific use case you have in mind.

Reply:
Usually we have one-paragraph texts, so as far as the usage, the extra brackets are not inconvenient, IMO.
E.g.:
```
errorText={[getLocale('redirectModalWalletOwnershipVerificationFailureText').replace('$1', getWalletProviderName(externalWallet))]}
errorTextLink={['https://community.brave.com']}
```
Now that we introduce multi-paragraph texts, I'm pretty sure we'll need a means of setting a separate link for each of them sooner or later. 🙂 That said, I can get rid of it if you find it unnecessary.

Reply:
OK, gotcha. It's not a deal-breaker for me by any stretch. The only reason I brought it up is that this construction seems less than ideal (and would probably be the norm for the majority of these cases, I think):

https://github.com/brave/brave-core/pull/12456/files#diff-0c26c120bbe64c0639795a351794bad4c74ef608299086d608006fdfc7d320e6R239-R243

Reply:
Oh, I see. I think we can clean up the syntax there a bit:
```
export interface Props {
  ...
  errorTextLink?: (string | undefined)[]
  ...
}

...

errorTextLink={[,,,'https://support.brave.com/hc/en-us/articles/4494596374925']}
```
WDYT?

Reply:
Could it be `errorTextLink?: string | string[]`? I feel like that may be the simplest approach, assuming it's possible and doesn't over-complicate the code. @zenparsing might have some ideas here.

Reply:
So the current (undocumented) protocol here is that:

- `$1` is the provider name
- `$2` is the start of the learn more link
- `$3` is the end of the learn more link

My guess is that the fact that we have more than one line now doesn't really change this protocol, and we should probably just support the a single learn more link (which can be in any of the lines).

So my vote would be to leave it `errorTextLink?: string`.

Reply:
Amended.